### DISCUSSION TITLE
SSL peer certificate was not OK

### DISCUSSION THREAD

**Discussion by HerambMahajan** [idaholab/moose#20058] [Q&A General] [answered]
During MOOSE Conda packages installation, 'mamba install moose-tools moose-libmesh' command gave me following error: 
" SSL peer certificate or SSH remote key was not OK. [https://conda.software.inl.gov/public/noarch/repodata.json] SSL certificate problem: unable to get local issuer certificate. ". 
How can I connect my machine to license servers?

**Comment by milljm** [comment, marked as answer]
We have seen this error most frequent when the user is using some sort of VPN software, or possibly has proxy settings set. Proxies that allows that person's corporation to perform MITM (man in the middle) inspections on the data being transferred to and from machine. These inspections put the package back together, but not in a way which most terminal application trust (corporation using self-signed root/authority certificates).

You can try the following to 'ignore' SSL errors. But this allows for additional security concerns.

```bash
conda config --set ssl_verify false
```
And then try your `mamba` command again.

Once complete, if you wish to be more secure once again:
```bash
conda config --set ssl_verify true
```

**Reply by HerambMahajan** [reply]
Thank you. That worked!
